testsuite/tests/parallel/catch_break.ml

```diff
@@ -23,43 +44,65 @@ Success.
 
 *)
 
+let delay = 0.001 (* 1 ms *)
+let fuel = Atomic.make 1000 (* 1s *)
+
 let print = if verbose then print_endline else fun _ -> ()
```

Comment:
Why the conditional printing? Do we lose something by printing all the time, with a reference file? (Maybe there is an issue with printing in the buggy cases?)

Reply:
The output is unreliable, see the comment above it
```ocaml
(* Expected when verbose (depending on scheduling and platform):
```